gulp/tasks/img.js

```diff
@@ -1,5 +1,5 @@
-const gulp = require("gulp");
+const gulp = require('gulp');
```

Comment:
Why do you have single quotes there? Prettier should format them. Guess, you have the problem due to lack of `.prettierrc` config file